### ISSUE TITLE
one mistake in UIWebView+AFNetWorking category

### ISSUE THREAD

**Issue by juvham** [AFNetworking/AFNetworking#3446]
in   
```
-(void)loadRequest:(NSURLRequest *)request
           MIMEType:(NSString *)MIMEType
```
this method in success block
```
                if ([strongSelf.delegate respondsToSelector:@selector(webViewDidStartLoad:)]) {
                    [strongSelf.delegate webViewDidFinishLoad:strongSelf];
                }
```
this line I think it's a mistake

**Comment by tsheaff** [comment]
Yes I agree @mattt who is the lead maintainer of this project now?

This causes delegate method to be called twice for me, first by AFNetworking then again by the system.

**Comment by tsheaff** [comment]
Oh, I may have missed @juvham's intent with this issue. Are you complaining that `@selector(webViewDidStartLoad:)` should have been `@selector(webViewDidFinishLoad:)`.

If so https://github.com/AFNetworking/AFNetworking/issues/3533 is a dupe of this issue.

method being called twice seems like a separate issue.

**Comment by kcharwood** [comment]
The typo is addressed in #3727 

@tsheaff Would you be able to put together a test case showing the double callback?

**Comment by tsheaff** [comment]
I probably won't have time this week to do so
